### DISCUSSION TITLE
What is this called and what's causing it and and do I fix it (please)?

### DISCUSSION THREAD

**Discussion by neurodiverseEsoteric** [obsproject/obs-studio#6083] [General]
# PHOTO-SENSITIVITY WARNING!

https://user-images.githubusercontent.com/577091/156816190-50f09953-49c6-437a-8c91-f375fa243962.mp4

**Comment by sebastian-s-beckmann** [comment]
This discussion appears to be a support request, for which we don't use GitHub Discussions.
Please make sure to visit the support sections on our [Discord server](https://discord.gg/obsproject) or [Forums](https://obsproject.com/forum).

Thank you!

**Reply by neurodiverseEsoteric** [reply]
I'd have to decimate the bit rate to get the video uploaded on Discord, I have too many user/pass to manage for forums already, and I don't even know how to describe the issue, and support requests tend to want written specifics...

**Reply by sebastian-s-beckmann** [reply]
> I'd have to decimate the bit rate to get the video uploaded on Discord

Our server is verified and boosted to level 3, anyone can upload 50MB videos

> I don't even know how to describe the issue, and support requests tend to want written specifics...

That's... the point. People need to know specifics to be able to help you. Just from seeing the video I'd have no idea what's going on. Also, a Discord message can be more casual than a Forum post.

Either way, you won't be getting support here.
